Doc/whatsnew/3.11.rst

```diff
@@ -101,9 +101,9 @@ that caused the error instead of just the line. For example:
                                ^^^^^^^^^
     AttributeError: 'NoneType' object has no attribute 'x'
 
-Previous versions of the interpreter would point to just the line making it
+Previous versions of the interpreter would point to just the line, making it
 ambiguous which object was ``None``. These enhanced errors can also be helpful
-when dealing with deeply nested dictionary objects and multiple function calls,
+when dealing with deeply nested :class:`dict` objects and multiple function calls:
 
 .. code-block:: python
```

Comment:
Can't we use this here?
```suggestion
when dealing with deeply nested :class:`dict` objects and multiple function calls::
```

Reply:
I didn't see a strong reason to change this from the original source, since it renders equivalently, is a little more explicit and (IIRC) warns if the syntax used doesn't actually lex according to the declared lexer (vs. `::` fails silently). But I can if you think its important